### DISCUSSION TITLE
Q&A: Image Compression Settings

### DISCUSSION THREAD

**Discussion by Glory2Antares** [image-rs/image-png#481] [Q&A] [answered]
Hello, I was wondering what the best compression setting offered by this crate is. Right now I'm using `encoder.set_compression(png::Compression::Best);` to compress my png's. 

This works well but I noticed in the [reference ](https://docs.rs/png/latest/png/enum.Compression.html)it says:  

> **Best**
> Higher compression level
> Best in this context isn’t actually the highest possible level the encoder can do, but is meant to emulate the Best setting in the Flate2 >library.

This seems to suggest there is a higher level of compression the encoder can do...and that I would like to use, but I'm unsure as to how to access it. 

Any help is greatly appreciated!

**Comment by fintelia** [comment, marked as answer]
Uh, I don't think that line is accurate anymore. Picking Best ends up using flate2 with compression level 9 which is the highest. If you want better compression that than, I'd suggest either looking at a PNG optimizer like oxipng or else a different image format

**Reply by Glory2Antares** [reply]
Thanks for the reply. I tried oxipng and it works well.
